### ISSUE TITLE
group and groups resource: members does not work when the group has multiple members

### ISSUE THREAD

**Issue by steversmith** [inspec/inspec#4342]
### 🎛 Description

This is a 🐛 bug report

group and groups resource: members does not work when the group has multiple members

### 🌍 InSpec and Platform Version
Redhat Linux 6.6 running via the test kitchen:
inspec version: 3.9.3
kitchen version: 1.25.0
kitchen-inspec: 1.1.0

### 🤔 Replication Case

Using a simple cookbook recipe, I created a VM with two test groups:
test1 has one member, userA
test2 has two members, userA and userB
```
grep test /etc/group
test1:x:509:userA
test2:x:510:userA,userB
```

I setup two control groups for testing: both controls should pass, but Group test2 fails.
```
control 'Group test1' do
  describe group('test1') do
    its("members") { should include 'userA' }
    its("members") { should be_in ['userA'] }
  end

  describe groups.where { name == 'test1' } do
    its("members") { should include 'userA' }
    its("members") { should be_in ['userA'] }
  end
end

control 'Group test2' do
  describe group('test2') do
    its("members") { should include 'userA' }
    its("members") { should include 'userB' }
    its("members") { should be_in ['userA', 'userB'] }
  end

  describe groups.where { name == 'test2' } do
    its("members") { should include 'userA' }
    its("members") { should include 'userB' }
    its("members") { should be_in ['userA', 'userB'] }
  end
end
```

### 💁 Possible Solutions

I think that the root cause is that:
group.members returns a CSV formatted String of the members
groups.members returns a single element array with CSV formatted String of the members

### 💻 Stacktrace
The test output produces:
```
[PASS]  Group test1: Group test1
  [PASS]  Group test1 members should include "userA"
  [PASS]  Group test1 members should be in "userA"
  [PASS]  Groups with name == "test1" members should include "userA"
  [PASS]  Groups with name == "test1" members should be in "userA"
[FAIL]  Group test2: Group test2 (4 failed)
  [PASS]  Group test2 members should include "userA"
  [PASS]  Group test2 members should include "userB"
  [FAIL]  Group test2 members should be in "userA" and "userB"
  expected `userA,userB` to be in the list: `["userA", "userB"]`
  [FAIL]  Groups with name == "test2" members should include "userA"
  expected ["userA,userB"] to include "userA"
  [FAIL]  Groups with name == "test2" members should include "userB"
  expected ["userA,userB"] to include "userB"
  [FAIL]  Groups with name == "test2" members should be in "userA" and "userB"
  expected `["userA,userB"]` to be in the list: `["userA", "userB"]`
  Diff:
    ["userA,userB"]
```

### Possible Workaround
A workaround is to use the **etc_group** resource instead:
```
control 'etc_group test1' do
  describe etc_group.where(name: 'test1') do
    its("users") { should include 'userA' }
    its("users") { should be_in ['userA'] }
  end
end

control 'etc_group test2' do
  describe etc_group.where(name: 'test2') do
    its("users") { should include 'userA' }
    its("users") { should include 'userB' }
    its("users") { should be_in ['userA', 'userB'] }
  end
end
```

Produces 
```
[PASS]  etc_group test1: #<Inspec::Resources::EtcGroupView:0x000000000bef9650>
  [PASS]  #<Inspec::Resources::EtcGroupView:0x000000000bef9650> users should include "userA"
  [PASS]  #<Inspec::Resources::EtcGroupView:0x000000000bef9650> users should be in "userA"
[PASS]  etc_group test2: #<Inspec::Resources::EtcGroupView:0x000000000bf61548>
  [PASS]  #<Inspec::Resources::EtcGroupView:0x000000000bf61548> users should include "userA"
  [PASS]  #<Inspec::Resources::EtcGroupView:0x000000000bf61548> users should include "userB"
  [PASS]  #<Inspec::Resources::EtcGroupView:0x000000000bf61548> users should be in "userA" and "userB"
```

**Comment by asyd** [comment]
Any news?

**Comment by clintoncwolfe** [comment]
Fixed on #5479
